src/use-font-size.hook.js

```diff
@@ -14,8 +14,10 @@ export function useFontSize({defaultFontSize = '14px', fontSizes}) {
 
   return {
     currentlySelectedFontSize,
-    setSize(fontSize) {
-      const integerSize = fontSizes.findIndex(size => size === fontSize) + 1
+    setSize(fontSizeParam) {
+      const integerSize = fontSizeParam === fontSize
+        ? fontSizes.length + 1
+        : fontSizes.findIndex(size => size === fontSizeParam) + 1
```

Comment:
If we're already at that font size, let's set size integer to just one above the length of array. we will reserve length + 1 for styling for default

A potential problem here is the fontSize not being updated before `setSize()` is invoked by user click. It is set when selectionChangedLIstener triggers function selectionChanged(). I have confirmed the selectionChangedListener fires every time I click to adjust font and that newSize !== fontSize which enters the if block to invoke setFontSize.